src/Centreon/Domain/PlatformTopology/PlatformTopologyService.php

```diff
@@ -233,18 +237,22 @@ public function addPlatformToTopology(PlatformTopology $platformTopology): void
                 $baseApiEndpoint = $foundPlatformInformation->getApiScheme() . '://' .
```

Comment:
The end point of the API must be defined by the controller and not by the business.

Reply:
As you know, the payload and path to the API is defined using the data retrieved from the business
We'll discuss this and I'll correct it in another PR